inference/models/tests/models_test.py

```diff
@@ -0,0 +1,56 @@
+from sklearn.externals import joblib
+import sys
+sys.path.append("../../..")
+from inference.inference_network import InferenceNetwork
+import csv
+import os
+os.chdir('../../../')
+
+def test_accuracy():
+
+    model_test_map = {
+        "query_classifier.py" : "inference/models/tests/test_input.csv",
+    }
+
+    for model, test_path in model_test_map.items():
+
+
+        os.system('python3 ' + model) # train models, dumps into model .pkl file
```

Comment:
There is nothing wrong with this implementation, but I would suggest importing query classifier as a module so you won't be spawning a new process for every model. There are other instances where `os.system`  is being used in the repository, and I have been meaning to update those to be modules as well 😅 . This script will likely be used when performing pre-deployment checks so it would be a good idea to be efficient with compute.

It would look something like this:
```python
import query_classifier

query_classifier.main()
```